pages/recording/[id]/share.tsx

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
+import { useDispatch } from "react-redux";
+import { setModal } from "ui/actions/app";
+import { useGetRecordingId } from "ui/hooks/recordings";
+
+export default function Share() {
+  const recordingId = useGetRecordingId();
```

Comment:
Can probably drop this since you're pulling the recording id from the query too.

Reply:
left it on just in case we have to handle slugs

Reply:
gotcha :+1: